### ISSUE TITLE
auth specification is ignored by radicale

### ISSUE THREAD

**Issue by caesarsamsi** [Kozea/Radicale#409]
So, I'm pretty sure someparts of the config.py file is respected but some parts aren't.

For example:
- the port number is completely ignored (I set it to :5000 but it still reports on --debug that it's listening on 5000
- the auth keyword is completely ignored (I did not see any auth activity on --debug, and no error message is shown even though I've speficied FOOBAR for auth. I also randomly placed FOOBAR in auth.py file.

What's causing this and fix?

**Comment by liZe** [comment]
By `config.py`, you mean `config`, right?

**Comment by caesarsamsi** [comment]
No, I mean the config.py file. Since then I've found the /etc/radicale/config.py file was ignored completed. I had to use radicale --config instead of just radicale.

I've completely uninstalled radicale and then completely installed and using the config.py in the radicale installation directory, enabled SSL, enabled auth=htpassword, enabled rights=owner_only.

Then it all worked. I think it only requests auth when rights=owner_only.
